### ISSUE TITLE
Can't find a codec for class org.apache.logging.log4j.mongodb4.MongoDb4DocumentObject

### ISSUE THREAD

**Issue by MentosL** [apache/logging-log4j2#1439]
As clearly indicated in [the Log4j support page](https://logging.apache.org/log4j/2.x/support.html#issues), **please use mailing lists for questions!**

Issues asking questions will be removed, and you will be asked to post questions to the mailing lists instead.

when i use log4j2 to record log file，but  i could't record，occur exception ：

![image](https://user-images.githubusercontent.com/33775732/233314728-5ef80a0a-f6dd-4b15-9f57-19311dba0dc5.png)

```xml
      <dependency>
        <groupId>org.apache.logging.log4j</groupId>
        <artifactId>log4j-mongodb4</artifactId>
        <version>2.17.1</version>
      </dependency>
      <dependency>
        <groupId>org.apache.logging.log4j</groupId>
        <artifactId>log4j-mongodb4</artifactId>
        <version>2.6.3</version>
      </dependency>
```

i want know how to fix it  .     thanks

**Comment by ppkarwasz** [comment]
This is a duplicate of [LOG4J2-3392](https://issues.apache.org/jira/browse/LOG4J2-3392), which was solved in version 2.17.2.

**Comment by mpenicaud** [comment]
issue also occurs with version 3.0.0-alpha1

**Comment by ppkarwasz** [comment]
Fixed regression in #2164
